src/main/java/rs/hybridit/serviceImpl/LibraryServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package rs.hybridit.serviceImpl;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import rs.hybridit.model.Library;
+import rs.hybridit.repository.LibraryRepository;
+import rs.hybridit.service.LibraryService;
+
+import java.util.List;
+
+@Service
+public class LibraryServiceImpl implements LibraryService {
+    @Autowired
+    LibraryRepository libraryRepository;
+
```

Comment:
Instead of autowired, inject trough constructor

private final LibraryRepository libraryRepository;

public LibraryServiceImpl(LibraryRepository libraryRepository) {
this.libraryRepository = libraryRepository;
}